util/gen_action_info.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python
+
+import csv
+import json
+import os
+import urllib.request
+import re
+import io
+from collections import defaultdict
+
+"""Action data is available in csv form that has 3 headers looks like:
```

Comment:
Linking to the csv directly might be more clear than a large chunk of text?

Reply:
I agree.  My initial thought that there would still be that record of what the script is expecting so that it would be a little easier to debug in the event of any structure changes.

That being said, when I put that in there I was also unhappy with the end result.